server/src/main/java/io/deephaven/server/auth/TrivialAuthContextProvider.java

```diff
@@ -19,6 +27,11 @@ public boolean supportsProtocol(final long authProtocol) {
 
     @Override
     public AuthContext authenticate(final long protocolVersion, final ByteString payload) {
+        List<X509Certificate> x509Certificates = MTlsCertificate.CLIENT_CERTIFICATES.get();
+        if (x509Certificates != null && !x509Certificates.isEmpty()) {
+            log.info().append(x509Certificates.get(0).getSubjectDN().toString()).endl();
```

Comment:
i put this in for debugging, do we want to land it as-is?

Reply:
I imagine we can take it out - we can create a more appropriate `AuthContext` for mTLS as the authorization gets fleshed out.

Reply:
Or maybe, keep a specific AuthContextProvider w/ mTLS example - that may be better?...

Reply:
Yep, seems reasonable either way - maybe kick the log level down to debug/trace in the meantime if you leave it in?

Reply:
I've removed it for now.